run_llama_train.sh

```diff
@@ -7,9 +7,6 @@
 
 set -ex
 
-# libUV is a scalable backend for TCPStore which is used in processGroup
-# rendezvous. This is the recommended backend for distributed training.
-export USE_LIBUV=1
 TRAINER_DIR=${TRAINER_DIR:-/home/$USER/local/torchtitan}
```

Comment:
May I ask what this line is doing?
My repo sits in `/home/$USER/torchtitan`, would it be a problem?

Reply:
i don't think this line is doing anything at all.  I actually have a later PR to delete it.